File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int n = nums.size();
        int count = 0 ;
        int res= 0 ;
        for(int i=0;i<n;i++){
            if(nums[i]==0){
                count= 0 ;
            } else{
                count++;
                res = max(count,res);
            }
        }
        return res;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        n =  len(nums)
        count = 0
        res = 0 
        for i in range(n):
            if nums[i]==0:
                count =0
            else:
                count += 1
                res = max(count , res)
        return res
        